### DISCUSSION TITLE
poolController stops after only 3-4 minute work

### DISCUSSION THREAD

**Discussion by ivanvach** [tagyoureit/nodejs-poolController#657] [App and O/S Config Questions]
I have serious problem. The poolController stops with this error after only 3-4 minutes of working

Error: read ECONNRESET
    at TCP.onStreamRead (node:internal/stream_base_commons:217:20)

Pls. help

**Comment by tagyoureit** [comment]
responded here: https://github.com/parnic/node-screenlogic/issues/61#issuecomment-1356326406
